test/latest/azuredisk-csi-driver/templates/disk.csi.azure.com_azvolumeattachments.yaml

```diff
@@ -85,11 +85,39 @@ spec:
               error:
                 description: Error occured during attach/detach of volume
                 properties:
+                  currentNode:
```

Comment:
Where are these changes from? It seems like changes from `v1alpha1/types.go` were not added to the commit. And are these fields used anywhere by the crdprovisioner or the controllers?

Reply:
https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/blob/73f336b4a0cb7e4a279593ed790049b5cccacc29/pkg/apis/azuredisk/v1alpha1/types.go#L181

Looks like this line bringing in the change.

Reply:
Yes, its being used in the provisioners to handle the DanglingAttachError.